watchtower-plugin/src/retrier.rs

```diff
@@ -647,13 +660,17 @@ mod tests {
         add_appointment_receipt.sign(&tower_sk);
         let add_appointment_response =
             get_dummy_add_appointment_response(appointment.locator, &add_appointment_receipt);
-        let api_mock = server.mock(|when, then| {
-            when.method(POST).path(Endpoint::AddAppointment.path());
-            then.status(200)
-                .delay(Duration::from_secs_f64(API_DELAY))
-                .header("content-type", "application/json")
-                .json_body(json!(add_appointment_response));
-        });
+
+        let api_mock = server
+            .mock("POST", Endpoint::AddAppointment.path().as_str())
+            .with_status(200)
+            .with_header("content-type", "application/json")
+            .with_body_from_request(move |_| {
+                std::thread::sleep(Duration::from_secs_f64(API_DELAY));
```

Comment:
We imported `std::thread` in the tests, let's omit the `std::` namespace.

Reply:
That must have been vscode on its own, I'll remove the `std::thread` import since that'll create a smaller diff.